.devcontainer/devcontainer.json

```diff
@@ -15,6 +15,10 @@
 		"terminal.integrated.defaultProfile.linux": "zsh"
 	},
 
+	"hostRequirements": {
+		"cpus": 4
```

Comment:
Upgrading host requirement since we're now running emulator and nginx containers.